tests/plotting/test_plotting.py

```diff
@@ -2786,7 +2658,9 @@ def test_plot_composite_poly_component_single(multiblock_poly):
     pl.show()
 
 
-def test_plot_composite_poly_component_nested_multiblock(multiblock_poly):
+def test_plot_composite_poly_component_nested_multiblock(multiblock_poly, verify_image_cache):
```

Comment:
Is there a way we can autouse verify_image_cache for an entire file? Seems like it would be possible and would eliminate a lot of work maintaining function signatures.

Reply:
We are doing that in lines 92-94, in this test we need to use an attribute of the plugin. We can move the skip attributes back to the lists, as in lines 81-130, but I though it would be easier to read and maintain this way.

Reply:
I like the implentation in this PR.  It means that we annotate at the point of use.  When reading a test it is immediately obvious if it is a high variance test for example.  One downside is that you have to do 'ctrl-f' or similar to find all usages like this.  I wonder if there is a way to utilize decorators for this purpose, but this is for another PR.

```py
@skip_windows_cache
def test_plot_1():
    ...
```

Reply:
Let's leave this for another PR, but I appreciate (and agree) with this.